solarlog/plugin.yaml

```diff
@@ -21,6 +21,14 @@ plugin:
 
 parameters:
     # Definition of parameters to be configured in etc/plugin.yaml (enter 'parameters: NONE', if section should be empty)
+    fw:
```

Comment:
Did you check the plugins metadata with ``python3 tools/plugin_metadata_checker.py -c solarlog``
Probably an attribute fw (firmware) could be a list of existing firmware flavours?

Reply:
I ran the check and removed the mandatory because it's not allowed in connection with default. Also, I renamed the attribute to fw2x which is probably not used anywhere else.